test/collection_test.js

```diff
@@ -446,6 +446,36 @@ describe("Collection", () => {
       return articles.update({id: "deadbeef"})
         .should.be.rejectedWith(Error, /Invalid Id/);
     });
+
+    it("should remove previous record fields", () => {
+      return articles.create(article)
+        .then(res => articles.get(res.data.id))
+        .then(res => {
+          return articles.update(
+            Object.assign({}, {id: res.data.id}, {
+              title: "new title",
+            }));
+        })
+        .then(res => res.data)
+          .should.eventually.not.have.property("url");
+    });
```

Comment:
Adding this extra test is maybe more related to #310 than to this PR itself, but guess it can't hurt.